### ISSUE TITLE
getVersion() returns wrong array key 

### ISSUE THREAD

**Issue by ivpfals** [php-memcached-dev/php-memcached#290]
Hi,
I'm using memcached 3.0.0b1 with libmemcached 1.0.16/Memcached 1.4.15 under PHP 7.0.14. 
If you are calling the getVersion() method the array key returned for the first added server is wrong, it contains the wrong port number:

```
$MC = new Memcached;
$MC->addServer('localhost','11211');

$version = $MC->getVersion();
var_dump($version);

$stats = $MC->getStats();
var_dump($stats);
```
The var_dump($version) returns:

```
array(1) {
  ["localhost:11210"]=>
  string(6) "1.4.15"
}

```
while the var_dump($stats) returns:

```
array(1) {
  ["localhost:11211"]=>
  array(48) {
    ["pid"]=>
    int(10346)
    ["uptime"]=>
    int(3521)
    ["time"]=>
    int(1484048172)
    ["version"]=>
    string(6) "1.4.15"
    ["libevent"]=>
    string(13) "2.0.21-stable"
    ...
    ...
   
```
Seems that getVersion() cuts somehow the port number?

**Comment by sodabrew** [comment]
Confirmed. Isn't that the darndest thing.

**Comment by sodabrew** [comment]
I think the original bug was just a mistake. The same mistake was made for getStats originally, and was corrected in cbcbece3619bd067d64094cc04f7d6a676a28b66. I will fix this shortly in the master branch.

**Comment by ivpfals** [comment]
Hi,

just tested the updated code and now everything is fine.

Thanks for your help!
